samples/subsys/audio/sof/prj.conf

```diff
@@ -21,3 +21,5 @@ CONFIG_SMP_BOOT_DELAY=y
 # Fix the sys ticks value  until following bugs are solved:
 # - https://github.com/zephyrproject-rtos/zephyr/issues/46378
 CONFIG_SYS_CLOCK_TICKS_PER_SEC=15000
+CONFIG_DAI=y
+CONFIG_HEAP_MEM_POOL_SIZE=2048
```

Comment:
Yeah, twister runs building SOF application for every cavs platform so if they are green this must be working.
We turned of ace board from twister because we can't resolve Kconfig dependencies pretty well but rest of the platforms are fine.

Reply:
The GitHub CI does not run anything on audio hardware. Have you tested it to make sure this doesn't break non-MTL platforms? If not, I recommend only enabling these kconfigs in board overlay.